File: Web/RMA-Web-Shell/projects/penscare/src/app/shared-penscare/models/sftp-request.ts

import { SftpRequestStatusTypeEnum } from "projects/shared-models-lib/src/lib/enums/sftp-request-status-type-enum";
import { SftpRequestTypeEnum } from "projects/shared-models-lib/src/lib/enums/sftp-request-type-enum";

export class SftpRequest {
    sftpRequestId: number;
    sftpRequestType: SftpRequestTypeEnum;
    fileName: string;
    sftpRequestStatus: SftpRequestStatusTypeEnum;
    itemsInRequest?: number;
    noResponses: number;
    requestedDate: Date;
    isDeleted: boolean;
    createdDate: Date;
    createdBy: string;
    modifiedDate: Date;
    modifiedBy: string;
}

File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/SftpRequest.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class SftpRequest
    {
        public int SftpRequestId { get; set; }
        public SftpRequestTypeEnum SftpRequestType { get; set; }
        public string FileName { get; set; }
        public SftpRequestStatusEnum SftpRequestStatus { get; set; }
        public int? ItemsInRequest { get; set; } 
        public int NoResponses { get; set; } 
        public System.DateTime RequestedDate { get; set; }
        public bool IsDeleted { get; set; } 
        public System.DateTime CreatedDate { get; set; } 
        public string CreatedBy { get; set; } 
        public System.DateTime ModifiedDate { get; set; } 
        public string ModifiedBy { get; set; } 

    }
}